_saas-integrations/ebay/v1/ebay-v1.md

```diff
@@ -0,0 +1,160 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: eBay (v1)
+permalink: /integrations/saas/ebay ## Add if there are multiple versions: /vVERSION
+keywords: ebay, integration, schema, etl ebay, ebay etl, ebay schema
+layout: singer
+# input: false
+
+key: "ebay-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "ebay"
+display_name: "eBay"
+
+singer: true
+status-url: "https://www.ebay.com/sts"
+
+tap-name: "eBay" ## Ex: Intercom, not intercom
+repo-url: https://github.com/singer-io/tap-ebay
+
+this-version: "1"
+
+api: |
+  [{{ integration.display_name }} Fulfillment API](https://developer.ebay.com/api-docs/sell/fulfillment/static/overview.html){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: false 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Standard"
+
+api-type: "platform.ebay"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**An {{ integration.display_name }} seller account**. This account owns the sales data."
+  - item: "**An {{ integration.display_name }} developer account** to allow Stitch to access the sales data."
```

Comment:
Perhaps link to the page where a user can sign up for a developer account. Something like

> **An {{ integration.display_name }} developer account**. This is required to allow Stitch to access the sales data in your {{ integration.display_name }} seller account. If you don't have a developer account, you can sign up for one [here](https://link-to-ebay-dev-account-signup.com){:target="new"}.